x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_related_list.tsx

```diff
@@ -34,16 +36,56 @@ interface MatchingEventEntry {
   setQueryParams: () => void;
 }
 
+const StyledRelatedLimitWarning = styled(RelatedEventLimitWarning)`
+  flex-flow: row wrap;
+  display: block;
+  align-items: baseline;
+  margin-top: 1em;
+
+  & .euiCallOutHeader {
+    display: inline;
+    margin-right: 0.25em;
+  }
+
+  & .euiText {
+    display: inline;
+  }
+
+  & .euiText p {
+    display: inline;
+  }
+`;
+
 const DisplayList = memo(function DisplayList({
   crumbs,
   matchingEventEntries,
+  eventType,
+  processEntityId,
+  aggregateCountForEventType,
 }: {
   crumbs: Array<{ text: string | JSX.Element; onClick: () => void }>;
   matchingEventEntries: MatchingEventEntry[];
+  eventType: string;
+  processEntityId: string;
+  aggregateCountForEventType: number;
 }) {
+  const relatedEventResponsesById = useSelector(selectors.relatedEventsByEntityId);
+  const responseForThisNode = relatedEventResponsesById.get(processEntityId);
+  const shouldShowLimitWarning =
+    responseForThisNode &&
+    responseForThisNode.nextEvent !== null &&
+    matchingEventEntries.length < aggregateCountForEventType;
+
   return (
     <>
       <StyledBreadcrumbs breadcrumbs={crumbs} />
+      {shouldShowLimitWarning ? (

```

Comment:
you can just make this `&&` and remove the ternary option. `{shouldShowLimitWarning && <Styled... />}`